### DISCUSSION TITLE
Custom logs directory?

### DISCUSSION THREAD

**Discussion by Marco-Lugo** [tnodir/fort#201] [Q&A] [answered]
It would be helpful to have the option to choose the directory where your firewall saves its logs.
Thanks.

**Comment by tnodir** [comment, marked as answer]
You can use:
- `--logs <PATH>` command line argument
- or better, rename the `FortFirewall.exe.example.ini` to `FortFirewall.exe.ini` in the app's installed directory and set `logsDir`
